foolsgold/static/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>
-      FoolsGold Ruby Rack in Rust Server Extravaganza
+      Ferrocarril Ruby Rack in Rust Server Extravaganza
```

Comment:
Ditto ferrocarril is the workspace and eventual rails demo. Fools gold is a separate proof of concept.